src/_articles/libraries/serialization.md

```diff
@@ -6,8 +6,22 @@ written: 2015-02-09
 category: libraries
 ---
 
+<aside class="alert alert-info" markdown="1">
+**Note:** While this article still embodies the correct approach for
+adding serialization to your app,
```

Comment:
"embodies the correct approach for adding" doesn't sound right to me (and a bit fancy :). Also, we might want to shorten this. Maybe:

This article is a bit outdated. Its general information and approach are still useful,
but it doesn't cover new software such as the
[built_value package](...).
For more up-to-date information and discussion of Dart serialization solutions, see the unofficial repo
[matanlurey/dart_serialize_proposal](https://github.com/matanlurey/dart_serialize_proposal).